eventmesh-common/src/main/java/org/apache/eventmesh/common/config/CommonConfiguration.java

```diff
@@ -72,6 +72,8 @@ public class CommonConfiguration {
     @ConfigFiled(field = "connector.plugin.type", notEmpty = true)
     private String eventMeshConnectorPluginType = "rocketmq";
 
+    @ConfigFiled(field = "security.validation.type.token", notEmpty = true)
+    private boolean eventMeshSecurityValidateTypeToken = false;
```

Comment:
this configuration need add to the eventmesh.properties file, since it is marked with @ConfigFiled, you can refer to `eventMeshConnectorPluginType`

Reply:
OK, I will fix it next time I submit